### DISCUSSION TITLE
Combobox with required labelText

### DISCUSSION THREAD

**Discussion by LechK** [twilio-labs/paste#3047] [Ideas] [answered]
Hey Paste Team!
I'm just wondering, why Combobox has REQUIRED labelText? 
Sometimes it is really clear, what we want to select, so label is not needed.
Just asking :)

**Comment by nkrantz** [comment, marked as answer]
Hey @LechK - good question. The reason we require label text on our Combobox (and all input fields) is for accessibility reasons. Just because the intended actions on a Combobox may be really clear to you, that doesn't mean the same for all users. When something seems like it would be obvious without a label text, we're often relying on visual cues that not every user can see or understand. In order to ensure that each and every user will be able to interact with your form, whether or not they're using Assistive Technologies, keyboard/mouse, etc., it's important to always provide an associated label. Feel free to read more in [this blog article about why all inputs need labels](https://www.a11y-collective.com/every-input-needs-a-label/).

**Reply by LechK** [reply]
Thank You for answer!
